### PR TITLE
Disable kerberos in standalone

### PR DESCRIPTION
**Pull request by sreevatsanraman**
JIRA: https://issues.cask.co/browse/CDAP-7057
Build: http://builds.cask.co/browse/CDAP-RUT73

### CODE REVIEW

File: cdap-standalone/src/main/resources/cdap-site.xml [hunk 1]
```diff
@@ -197,4 +197,12 @@
     </description>
   </property>
 
+  <property>
+    <name>kerberos.auth.enabled</name>
+    <value>false</value>
+    <description>
+      Disable Kerberos authentication in standalone.
```
Comment (anwar6953) [inline cdap-standalone/src/main/resources/cdap-site.xml]: Description isn't supposed to be tied to the specific value we're configuring, right?

Probably just copy the description from cdap-default.xml - that's what the other properties in here seem to be doing.

Comment (sreevatsanraman) [inline cdap-standalone/src/main/resources/cdap-site.xml]: ok. Fixed.

**Comment by anwar6953** [comment]
lgtm

**Comment by sreevatsanraman** [comment]
Build passed merging.

**Pull request merged**